cmd/helm/install.go

```diff
@@ -310,7 +310,14 @@ func vals(valueFiles valueFiles, values []string) ([]byte, error) {
 	// User specified a values files via -f/--values
 	for _, filePath := range valueFiles {
 		currentMap := map[string]interface{}{}
-		bytes, err := ioutil.ReadFile(filePath)
+
+		var bytes []byte
+		var err error
+		if strings.Trim(filePath, " ") == "-" {
```

Comment:
There are more control characters like tabs, newline characters, carriage returns etc. that would be good to capture and trim which would be captured with `strings.TrimSpace`. But I also agree that this isn't a complete showstopper.